.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,17 @@
 \<your comments for this PR go here\>
 
+---
+
 Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md) lately? If not, do that first.
 
 I, the developer opening this PR, do solemnly pinky swear that:
 
 - [ ] I've followed [the instructions](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
-- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
-- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/bond/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing
 
 In all cases:
 
 - [ ] Get two thumbsworth of review and PO signoff if necessary. 
 - [ ] Verify all tests go green
 - [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
 - [ ] Test this change deployed correctly and works on dev environment after deployment
+- [ ] **Release to production** - Releasing to prod is a manual process (see [instructions to release to prod](https://github.com/DataBiosphere/bond/blob/develop/README.md#production-deployment-checklist)). Either do this now or create a ticket and schedule the release.
```

Comment:
Big plus 1 for creating a release ticket.  I think we talked about doing this last time, but we haven't had enough practice yet to build this habit.  We'll probably _still_ forget, but let's keep reminding ourselves.